kit/ForKit.cpp

```diff
@@ -480,21 +480,21 @@ int main(int argc, char** argv)
     /*WARNING*/     else if (hasAnyCapability())
     /*WARNING*/     {
     /*WARNING*/         if (!checkLoolUser)
-    /*WARNING*/             std::cerr << "Security: --disable-lool-user-checking failed, loolforkit has some capabilities set." << std::endl;
+    /*WARNING*/             LOG_FTL("Security: --disable-lool-user-checking failed, loolforkit has some capabilities set.");
 
-    /*WARNING*/         std::cerr << "Aborting." << std::endl;
+    /*WARNING*/         LOG_FTL("Aborting.");
```

Comment:
Nice. LOG_FTL does in fact output to std::cerr as it was designed exactly for this case (error that results in aborting the process).